pkg/install/2-removebootstrap.go

```diff
@@ -63,6 +66,15 @@ func (i *Installer) removeBootstrap(ctx context.Context) error {
 		return err
 	}
 
+	{
+		cli, err := operatorclient.NewForConfig(restConfig)
+		if err != nil {
+			return err
+		}
+
+		i.updateConsoleBranding(ctx, cli)
```

Comment:
`updateConsoleBranding` accepts a clientset interface, so should be easier to mock. But we will need to reconsider several of bits in the package, once we start addming tests.